enterprise_subsidy/apps/subsidy/tests/test_models.py

```diff
@@ -99,6 +101,49 @@ def test_price_for_content_not_in_catalog(self):
         with self.assertRaises(ContentNotFoundForCustomerException):
             self.subsidy.price_for_content('some-content-key')
 
+    def test_reference_uniqueness(self):
+        """
+        Tests that not soft-deleted, non-internal-only subsidies
```

Comment:
nit/clarification: Is it worth having a test case that covers when a subsidy _is_ soft-deleted? At first glance, it's not immediately clear what would happen differently with this logic when a subsidy is soft-deleted, if anything.